charts/signoz/templates/otel-collector-metrics/deployment.yaml

```diff
@@ -73,11 +73,11 @@ spec:
           {{- end }}
           env:
             {{- include "snippet.clickhouse-env" . | nindent 12 }}
-            - name: MY_POD_IP
-              valueFrom:
-                fieldRef:
-                  apiVersion: v1
-                  fieldPath: status.podIP
+            - name: K8S_CLUSTER_NAME
+              value: {{ default .Values.global.clusterName .Values.clusterName }}
+            - name: OTEL_RESOURCE_ATTRIBUTES
+              value: host.name=signoz-host,os.type=linux,signoz.component=otel-collector-metrics
```

Comment:
Is `cluster.name=${K8S_CLUSTER_NAME}` left out here intentionally?

Reply:
no, it is not.

Nice catch.